jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/FilterSettingsManager.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.filters;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.jbpm.workbench.common.client.filters.saved.SavedFilter;
+import org.jbpm.workbench.df.client.filter.FilterSettings;
+import org.uberfire.mvp.Command;
+
+public interface FilterSettingsManager {
+
+    void loadSavedFilters(Consumer<List<SavedFilter>> savedFiltersConsumer);
+
+    FilterSettings createDefaultFilterSettingsPrototype();
+
+    FilterSettings createFilterSettingsPrototype();
+
+    void saveFilterIntoPreferences(FilterSettings filterSettings,
+                                   Command callback);
+
+    void removeSavedFilterFromPreferences(String key);
+
+    void getFilterSettings(String key,
+                           Consumer<FilterSettings> filterSettingsConsumer);
+
+    void resetDefaultSavedFilters(Consumer<List<SavedFilter>> savedFiltersConsumer);
+
+    FilterSettings getVariablesFilterSettings(String name);
```

Comment:
How about making this method _default_, if not all implementations will actually override it?

Reply:
+1